src/routes/jira/configuration/jira-configuration-get.test.ts

```diff
@@ -65,11 +66,13 @@ describe("Jira Configuration Suite", () => {
 			.get(`/app/installations/15`)
 			.reply(200, singleInstallation);
 
+		const gheServerEnabled = await booleanFlag(BooleanFlags.GHE_SERVER, false, jiraHost);
```

Comment:
I'd reckon we don't use booleanFlag to switch the unitTest result.
If this line always return true, we may just test the `true` path.

Reply:
Initially, the booleanFlag will be turned on for some instances only, at that time this test case would fail. We can revisit this later once the booleanFlag is true for all instances...

Reply:
I mean it is better if the unit test doesn't rely on booleanFlag at runtime. Maybe we can mock the booleanFlag, so that it always return true in your test case, (or always return false in your test case). 

something like `mock(booleanFlag).mockReturnValue(false)` <--- I may not get the syntax right, but you get the idea. Explicit stating the assumption in the unit test is better than rely on implicit assumption.

Reply:
Oh, yup, agreed!
booleanFlag has been mocked to always return true